### PR TITLE
SG-15718: fixes panel on Windows

### PR DESCRIPTION
**Pull request by jfboismenu**
The previous code caused flickering on Windows. Now we were computing the sizes ourselves, which solves the issue.

### CODE REVIEW

File: python/tk_desktop/command_panel/recent_list.py [hunk 1]
```diff
@@ -22,7 +22,6 @@ class RecentList(BaseIconList):
 
     def __init__(self, parent):
         super(RecentList, self).__init__(parent, QtGui.QHBoxLayout())
-        self.setLayout(self._layout)
```
Comment (jfboismenu) [inline python/tk_desktop/command_panel/recent_list.py:25]: This line was present in the base class.

File: python/tk_desktop/desktop_window.py [hunk 2]
```diff
@@ -1081,28 +1080,6 @@ def set_groups(self, groups, show_recents=True):
...
-
-    def _force_panel_resize(self):
-        """
-        Force the command panel area to recompute the size of
-        its children.
-        """
-        # On Windows, we need to force a geometry update
-        # and widget resize so we do not get a horizontal
-        # scroll in the command panel.
-        self.ui.command_panel_area.updateGeometry()
```
Comment (jfboismenu) [inline python/tk_desktop/desktop_window.py:1103]: This code was causing the flickering.

File: tests/run_command_panel.py [hunk 3]
```diff
@@ -92,7 +111,7 @@ def save(self, key, recents):
     def load(self, key):
         return {
             command: {"timestamp": datetime.datetime.now()}
-            for command, _, _, _, _, _, _ in list(commands)[0:3]
+            for command, _, _, _, _, _, _ in list(commands)[0:6]
```
Comment (pscadding) [inline tests/run_command_panel.py:114]: This line is very hard to read, I know you only upped a number, but I don't know why. It's picking the first six values out of a commands variable that I think was a dictionary  but is being cast to a list.

Comment (pscadding) [reply tests/run_command_panel.py:114]: and that contains 7 values to be unpack of which we only want the first one.

Comment (pscadding) [reply tests/run_command_panel.py:114]: feel free to ignore, but just reading this, it hard to understand in what way it is parsing the data structure.

File: tests/run_command_panel.py [hunk 4]
```diff
@@ -132,19 +152,18 @@ def add_button():
...
-                    "from PySide2 import QtWidgets; QtWidgets.QApplication([]).exec_()",
-                ]
-            )
+            # subprocess.Popen(
+            #     [
+            #         sys.executable,
+            #         "-c",
+            #         "from PySide2 import QtWidgets; QtWidgets.QApplication([]).exec_()",
+            #     ]
+            # )
```
Comment (pscadding) [inline tests/run_command_panel.py:161]: Should this be commented out still?

**Pull request merged**